src/Logger/LoggerImpl.hpp

```diff
@@ -71,6 +71,10 @@ class LoggerImpl
     bool satellites_used_available;
     /** GPS Horizontal Dilution of precision */
     double hdop;
+    /** GPS Position (3D) Dilution of precision */
+    double pdop;
```

Comment:
Why copy this in the logger? The logger uses `hdop`, but doesn't use any of the new attributes (nor does any of your other commits add usage of it).